aries-site/src/examples/components/tag/CreateTag/CreateTag.js

```diff
@@ -0,0 +1,190 @@
+import React, { useContext, useEffect, useState } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Form,
+  FormField,
+  Select,
+  Tag,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { tagData } from '../data';
+import {
+  AppContainer,
+  getRegExp,
+  TagPageHeader,
+  prefix,
+  TagResults,
+  updateCreateOption,
+} from '.';
+
+const defaultNameOptions = [];
+const defaultValueOptions = [];
+const defaultName = '';
+const defaultValue = '';
+const defaultTags = [];
+
+Object.entries(tagData).forEach(([, value]) => {
+  defaultNameOptions.push({
```

Comment:
Per the user-generated aspect of this, we likely would not have a display name and thus this can simply push the key as a string.